poetry.lock

```diff
@@ -824,11 +824,11 @@ python-versions = "*"
 
 [[package]]
 name = "pillow"
-version = "8.4.0"
```

Comment:
Bumped pillow from 8.4.0 to 9.0.1 after Dependabot flagged the older version as a risk.